File: 프로그래머스/lv0/120813. 짝수는 싫어요/짝수는 싫어요.java
class Solution {
    public int[] solution(int n) {
        int[] answer = new int[(n+1) / 2];
        int index = 0;
        // answer 배열 안의 index를 초기화해주고
        for(int i = 1; i <= n; i++){
            if(i % 2 != 0){ // i 를 2로 나눴을 때 0이 아니면 홀수 이므로
                answer[index] =i;  // answer[index]에 넣는다.
                index++;  // 그리고 index를 1씩 증가시켜준다.
            }
        }   
        return answer;
    }
}

File: 프로그래머스/lv0/120813. 짝수는 싫어요/짝수는 싫어요.js
// 첫번째 방식
function solution(n) {
    let answer = []
                           
    let index = 0
    for(let i = 1; i <= n; i++){
        if(i % 2 != 0){
            answer[index] = i
            index++
        }
    }
    return answer
}

// 두번째 방식
function solution(n) {
    let answer = []
                           
    let element = 1
    for(let i = 0; i < Math.floor((n+1) /2); i++){
            answer[i] = element
            element += 2
        
    }
    return answer
}